message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1556,5 +1556,12 @@
       <field name="AOA" type="float" units="deg">Angle of Attack (degrees)</field>
       <field name="SSA" type="float" units="deg">Side Slip Angle (degrees)</field>
     </message>
+    <!-- wheel traveled distance message -->
+    <message id="11030" name="WHEEL_DISTANCE">
+      <description>Cumulative distances traveled by each wheel in meters (forward rotations increase these values, backward - decrease).</description>
+      <field type="uint64_t" name="time_usec" units="us">Timestamp (microseconds, synced to UNIX time or since system boot)</field>
```

Comment:
I think it should reuse the label and description wording of other messages, instead of adding a new label name

Reply:
The problem is that all the other messages are wrong :-). There is no need for name to include the units, and it shouldn't. We can't change that elsewhere without breaking compatibility, but no need to do it wrong on new messages. 

My 2 bits.

Reply:
@hamishwillee AFAIK both of us have removed all units from the descriptions.
What are the wrong messages?

Reply:
`name="time_usec"` should be `name="time"` or ...

Reply:
Yes, @hamishwillee I agree with you...... But to be consistent with other messages it should be kept at
name="time_usec"
It is bad to have a mix of "time" and "time_usec". And to correct all occurrences to "time_usec" would beak a lot of existing code.

So that is why I do think that the name "time_usec" should be kept, for consistency reasons. Some messages having "time", and some others having "time_usec" complicates code.

Reply:
> So that is why I do think that the name "time_usec" should be kept, for consistency reasons. Some messages having "time", and some others having "time_usec" complicates code.

I disagree, but happy to go with the majority.